docs/index.rst

```diff
@@ -2,42 +2,74 @@
 sunpy core Documentation
 ************************
 
-The sunpy core package is a community-developed, free and open-source solar
-data analysis environment for Python.
+``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
+
+
+New Users
+==========
+
+For help installing ``sunpy`` refer to our :ref:`installation guide <installing>`.
+
+
+.. todo::
```

Comment:
Are we ok with adding todos in files? 

Would an issue not be a better idea?

Reply:
I'm 👍 to opening an issue too, so we don't forget.